makefile

```diff
@@ -23,23 +23,27 @@ filenamesc := ft_strlen.c \
 			ft_strncmp.c \
 			ft_atoi.c \
```

Comment:
Aesthetic, and useful if there's one or more files you don't want to compile for some reason.
Much easier to comment out.